apyb/website/views.py

```diff
@@ -20,3 +21,12 @@ def get_context_data(self, **kwargs):
         context['hero_photos'] = HERO_PHOTOS
 
         return context
+
+class About(TemplateView):
+    template_name = 'website/markdown.html'
+
+    def get_context_data(self, **kwargs):
+        context = super(About, self).get_context_data(**kwargs)
+        markdown_content = open("website/markdown/about.md", 'r').read()
```

Comment:
That's a cool solution: Although I don't believe we'll use direct files in the future, it paves the way to make it simpler to load from a database, for example.

Nice work :+1:

Reply:
This way you are reprocessing something that is mainly static.

Loading from a database is better, but even better would be to format the markdown content only once and serve it as a static file.

Reply:
Yup. But, again, it proves that we can easily use Markdown for the pages.

One problem I see about creating the static pages is that, unless we use something ugly like iframes (well, _I_ think they are ugly, maybe there is something more direct, like server side imports), we need to generate the pages in the same style as the main page; if, for some reason, we update the style, all the pages should be regenerated. It seems we would just add complexity for something simple.

Another solution is still generate the page every time, but add a CDN in front of the site (disclaimer: I work at a CDN company :stuck_out_tongue: )

Reply:
From a performance point of view, the ideal solution would be to do both, cache it on a CDN, and regenerate whenever the style / content got updated.

(disclaimer: that's how we do it @ globo.com)